### DISCUSSION TITLE
Jax in embedded GPUs

### DISCUSSION THREAD

**Discussion by aizzaac** [jax-ml/jax#4503] [Q&A] [answered]
Hello

Has anyone tried using JAX with embedded GPUs? Like the Jetson Xavier?
The devboard can work in FP32, FP16 and INT8.

Thank you

**Comment by jekbradbury** [comment, marked as answer]
If you can successfully build a CUDA jaxlib on a Jetson board, I think we expect JAX would run fine. I don't think anyone's done it before, though.
